src/tools/web/test.sh

```diff
@@ -3,8 +3,7 @@ cd @CMAKE_CURRENT_SOURCE_DIR@
 
 echo " -~- Testing API elektra-web > elektrad -~- "
 cd elektrad
-npm install
-npm test
+go test ./...
```

Comment:
Is this executed on the build server?

Reply:
CMakeLists.txt contains this code:
```
if (DREDD_EXECUTABLE)

			# testing is possible
			add_test (NAME elektra-web COMMAND ./test.sh WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR})
			set_property (TEST elektra-web PROPERTY ENVIRONMENT "LD_LIBRARY_PATH=${CMAKE_BINARY_DIR}/lib")
		endif ()
```

But i'm pretty sure these tests are not executed because the PKG_CONFIG_PATH is not set and without it the tests can't be green.

Reply:
Yes, and you also need to make sure that dredd is installed in the Docker container.

Reply:
And with Docker container you mean Debian Buster + Stretch right?

Reply:
Buster would be enough.